.github/labels.md

```diff
@@ -0,0 +1,234 @@
+Issue occurs.  Categories share a prefix in the form `category_name:label_name`
+and are grouped together by color.  As a rule of thumb there is at least one
+label from every category that needs to be applied to an issue but use common
+sense to check if the label is approciate or not for the specific Issue.
+
+Currently there are three categories:
+
+
+#### Label category `category`
+
+Labels within this category describe is the intention of the Issue.  Usually
+there should be exact one intention for creating an Issue and it should not
+change during the lifetime of an Issue.
+
+#### Label category `component`
+
+The component lists the parts of the project this Issue affects or is affected
+by.  This should help developers to identify if they have required knowledge or
+interests to handle the issue.  There should be at least one component listed
+for an Issue.
+
+#### Label category `status`
+
+With the status category developers, managers or issue reporters can check if
+an issue needs further interaction and who should carry out this interaction.
+Issue have exactly one status.
+
+
+## Labels
+
+This section lists when an label should be applied and what actions may be taken
+additionally.
+
+
+#### Label `category:bug`
+
+The Issue describes or solves a percieved malfunction within the game.
+
+#### Label `category:feature`
+
+The Pull Request contains a new game feature.  Features are parts of the game
+which a player can interact with e.g.: A new species, a new gameplay mechanic
+or a new command line switch for the server that changes gameplay behaviour.
+
+#### Label `category:idea`
+
+??? No idea, overlaps with `category:feature` or `state:work in progress` IMO.
+
+#### Label `category:question`
+
+??? Questions don't belong into the issue tracker IMO.
+
+#### Label `category:refactoring`
+
+The Pull request contains a new or improved implementation of an existing
+feature or any other code restructuring that doesn't modify the current game
+mechanics or representation.
+
+#### Label `category:request`
+
+??? Questions don't belong into the issue tracker IMO.
+
+#### Label `category:tweak`
+
+??? No idea.
+
+#### Label `component:AI`
+
+The Issue requires knowledge about the python AI decision making code or affects
+it.
+
+#### Label `component:art music text assets`
+
+The Issue requires knowledge about creating game assets or verification if the
+assets match the style goals of the game.
+
+#### Label `component:build system`
+
+The Issue requires knowledge about CMake, Visual Studio, Xcode or the general
+build process.
+
+#### Label `component:common`
+
+??? No idea.
+
+#### Label `component:deployment`
+
+The Issue requires knowledge about the deployment process or the software used
+create the installer. ??? I see an overlap with `component:build system` here.
+
+#### Label `component:documentation`
+
+The Issue requires knowledge about Doxygen or primarly contains documentation.
+
+#### Label `component:game mechanic`
```

Comment:
@Vezzra 

`component` labels don't need to be distinct exclusive.  I don't see a problem with having a PR with "component:scripting, component:internal" and on top of that "component:game mechanic".  As I already wrote in the description the main purpose I see is that testing this PR requires probably at least one full game to check if the implementation works and not just open the game, check if interacting with a widget does the thing expected and close the game.